cmd/podman/runlabel.go

```diff
@@ -146,10 +147,26 @@ func runlabelCmd(c *cliconfig.RunlabelValues) error {
 		return err
 	}
 	if !c.Quiet {
-		fmt.Printf("Command: %s\n", strings.Join(cmd, " "))
+		fmt.Printf("command: %s\n", strings.Join(cmd, " "))
 		if c.Display {
 			return nil
 		}
 	}
+
+	// If container already exists && --replace given -- Nuke it
+	if c.Replace {
```

Comment:
wwyt about if c.Replace && util.StringInSlice(cmd, "--name") {

Reply:
Should we also verify that cmd[0] is podman (or /proc/self/bin or similar)?

Reply:
util.StringInSlice doesn't give me the index. I would need to iterate the slice twice if I used that.